### PR TITLE
Feat : add Typography cl

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
Looks good to me👍

**Review by prachi-sahani — COMMENTED** [review]
Code looks fine! Good to merge 👍🏻

**Review by prachi-sahani — APPROVED** [review]

File: docs/typography.html [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+                    <li>
+                        <a href="./../index.html"> Home</a>
+                    </li>
+                    <li>Docs</li>
+                </ul>
+                <div class="bold-600">Social</div>
+                <div>
+                    <ul class="bat-flex flex-gap-1 bat-social-links bat-list-non-bullets">
+                        <li class="bat-icon-social">
+                            <a class="link" href="https://github.com/ZuberDunge">
```
Comment (MaruthiPothuganti) [inline docs/typography.html:270]: Suggestion: have a target="_blank" for socials

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended a suggestion and overall Code looks Good👍

**Pull request merged**